test/model/adapters/sql/console_test.rb

```diff
@@ -0,0 +1,29 @@
+require 'test_helper'
+
+describe Hanami::Model::Adapters::Sql::Console do
+  describe 'deciding on which SQL console class to use, based on URI scheme' do
+    it 'mysql:// uri returns an instance of Console::Mysql' do
+      uri = 'mysql://username:password@localhost:1234/foo_development'
```

Comment:
I think we can replace `'username:password@localhost:1234/foo_development'` to let block. And after we can use this link with specific adapter i. e.:
``` ruby
require 'test_helper'

describe Hanami::Model::Adapters::Sql::Console do
  describe 'deciding on which SQL console class to use, based on URI scheme' do
    let(:url) { 'username:password@localhost:1234/foo_development' }

    it 'mysql:// uri returns an instance of Console::Mysql' do
      console = Hanami::Model::Adapters::Sql::Console.new("mysql://#{url}").send(:console)
      console.must_be_kind_of(Hanami::Model::Adapters::Sql::Consoles::Mysql)
    end

    it 'postgres:// uri returns an instance of Console::Postgresql' do
      console = Hanami::Model::Adapters::Sql::Console.new("postgres://#{url}").send(:console)
      console.must_be_kind_of(Hanami::Model::Adapters::Sql::Consoles::Postgresql)
    end

    # ....
  end
end
```

What do you think?